cp3_pipelines/2_segment_ilastik.cppipe

```diff
@@ -1,221 +1,249 @@
+    Fill holes in identified objects?:\xff\xfeY\x00e\x00s\x00
+    Threshold setting version:\xff\xfe9\x00
+    Threshold strategy:\xff\xfeG\x00l\x00o\x00b\x00a\x00l\x00
+    Thresholding method:\xff\xfeM\x00i\x00n\x00i\x00m\x00u\x00m\x00 \x00c\x00r\x00o\x00s\x00s\x00 \x00e\x00n\x00t\x00r\x00o\x00p\x00y\x00
+    Threshold smoothing scale:\xff\xfe1\x00.\x003\x004\x008\x008\x00
+    Threshold correction factor:\xff\xfe1\x00
+    Lower and upper bounds on threshold:\xff\xfe0\x00.\x000\x00,\x00 \x001\x00.\x000\x00
+    Manual threshold:\xff\xfe0\x00.\x000\x00
+    Select the measurement to threshold with:\xff\xfeN\x00o\x00n\x00e\x00
+    Two-class or three-class thresholding?:\xff\xfeT\x00w\x00o\x00 \x00c\x00l\x00a\x00s\x00s\x00e\x00s\x00
+    Assign pixels in the middle intensity class to the foreground or the background?:\xff\xfeF\x00o\x00r\x00e\x00g\x00r\x00o\x00u\x00n\x00d\x00
+    Size of adaptive window:\xff\xfe1\x000\x00
+    Lower outlier fraction:\xff\xfe0\x00.\x000\x005\x00
+    Upper outlier fraction:\xff\xfe0\x00.\x000\x005\x00
+    Averaging method:\xff\xfeM\x00e\x00a\x00n\x00
+    Variance method:\xff\xfeS\x00t\x00a\x00n\x00d\x00a\x00r\x00d\x00 \x00d\x00e\x00v\x00i\x00a\x00t\x00i\x00o\x00n\x00
+    # of deviations:\xff\xfe2\x00
+    Thresholding method:\xff\xfeO\x00t\x00s\x00u\x00
+
+RescaleObjects:[module_num:12|svn_version:\'Unknown\'|variable_revision_number:1|show_window:False|notes:\x5B\x5D|batch_state:array(\x5B\x5D, dtype=uint8)|enabled:True|wants_pause:False]
+    Select the input objects:\xff\xfeC\x00e\x00l\x00l\x00s\x00
+    Name the output objects:\xff\xfeR\x00e\x00s\x00c\x00a\x00l\x00e\x00d\x00C\x00e\x00l\x00l\x00s\x00
+    Select the operation:\xff\xfeD\x00o\x00w\x00n\x00s\x00c\x00a\x00l\x00e\x00 \x00o\x00b\x00j\x00e\x00c\x00t\x00s\x00 \x00w\x00i\x00t\x00h\x00 \x00a\x00 \x00s\x00c\x00a\x00l\x00i\x00n\x00g\x00 \x00f\x00a\x00c\x00t\x00o\x00r\x00
+    Factor to scale the object mask:\xff\xfe0\x00.\x005\x00
+
+OverlayOutlines:[module_num:13|svn_version:\'Unknown\'|variable_revision_number:4|show_window:False|notes:\x5B\'Outline of object \\xe2\\x80\\x9cNucFil\\xe2\\x80\\x9d is created as grayscale image. \'\x5D|batch_state:array(\x5B\x5D, dtype=uint8)|enabled:True|wants_pause:False]
+    Display outlines on a blank image?:\xff\xfeY\x00e\x00s\x00
+    Select image on which to display outlines:\xff\xfeN\x00o\x00n\x00e\x00
+    Name the output image:\xff\xfeR\x00e\x00s\x00c\x00a\x00l\x00l\x00e\x00d\x00C\x00e\x00l\x00l\x00s\x00O\x00u\x00t\x00l\x00i\x00n\x00e\x00
+    Outline display mode:\xff\xfeG\x00r\x00a\x00y\x00s\x00c\x00a\x00l\x00e\x00
+    Select method to determine brightness of outlines:\xff\xfeM\x00a\x00x\x00 \x00o\x00f\x00 \x00i\x00m\x00a\x00g\x00e\x00
+    How to outline:\xff\xfeI\x00n\x00n\x00e\x00r\x00
+    Select outline color:\xff\xfeR\x00e\x00d\x00
+    Select objects to display:\xff\xfeN\x00u\x00c\x00F\x00i\x00l\x00
```

Comment:
Is it intentional that you save the overlay of the `NucFil`?
Wouldn't saving the `RescaledCells` be more helpful?